ipaserver/install/dsinstance.py

```diff
@@ -670,7 +670,7 @@ def apply_updates(self):
             # very fatal errors only will raise exception
             raise RuntimeError("Update failed: %s" % e)
         installutils.store_version()
-
+        self.ldap_disconnect()
```

Comment:
Why it is located here, I don't see DS restart in this function, IMO this disconnect should be as close to restart as possible (i.e. in the same function as restart)